cdn/agent-loader/utils/configure.js

```diff
@@ -8,26 +8,20 @@ import { setConfiguration, setInfo, setLoaderConfig, setRuntime } from '@newreli
 let configured = false
 
 export function configure() {
-    return new Promise((resolve, reject) => {
-        if (configured) {
-            resolve(configured)
-            return
-        }
-        const nr = gosCDN()
-        
-        try {
+    if (configured) return
+    const nr = gosCDN()
+
+    try {
```

Comment:
Can we either move or create another try-catch around each of the lite/pro/spa entries? There's actually a low risk that something here goes wrong vs instrumenting & stageAggregator. Brought that up and this is prob a good time to put in such a trivial change that doesn't really require much testing. For the catch part, maybe attempt a sendBeacon to report that Agent ran into an error, then just silently quit. I think may be good idea to abort ee too.